roles/mongodb/templates/etc/mongodb/mongod.conf

```diff
@@ -0,0 +1,82 @@
+# mongod.conf
+
+# Where to store the data.
+
+# Note: if you run mongodb as a non-root user (recommended) you may
+# need to create and set permissions for this directory manually,
+# e.g., if the parent directory isn't mutable by the mongodb user.
+dbpath=/var/lib/mongodb
+
+#where to log
+logpath=/var/log/mongodb/mongod.log
+logappend=true
+
+pidfilepath = /var/run/mongodb.pid
+fork = true
```

Comment:
Currently, the "service mongod status/stop/restart" does not work. Please comment out the line "fork=true" to fix this issue.